File: reference-code/src/main/java/createcommands/arguments/types/scoreboard/TeamArguments.java
package createcommands.arguments.types.scoreboard;

import dev.jorel.commandapi.CommandAPICommand;
import dev.jorel.commandapi.arguments.TeamArgument;
import org.bukkit.scoreboard.Team;

class TeamArguments {
    static {
        // #region teamArgumentsExample
        new CommandAPICommand("togglepvp")
            .withArguments(new TeamArgument("team"))
            .executes((sender, args) -> {
                Team team = (Team) args.get("team");

                // Toggle pvp
                team.setAllowFriendlyFire(team.allowFriendlyFire());
            })
            .register();
        // #endregion teamArgumentsExample
    }
}

File: reference-code/src/main/kotlin/createcommands/arguments/types/scoreboard/TeamArguments.kt
package createcommands.arguments.types.scoreboard

import dev.jorel.commandapi.CommandAPICommand
import dev.jorel.commandapi.arguments.TeamArgument
import dev.jorel.commandapi.executors.CommandExecutor
import dev.jorel.commandapi.kotlindsl.anyExecutor
import dev.jorel.commandapi.kotlindsl.commandAPICommand
import dev.jorel.commandapi.kotlindsl.teamArgument
import org.bukkit.scoreboard.Team

fun teamArguments() {
    // #region teamArgumentsExample
    CommandAPICommand("togglepvp")
        .withArguments(TeamArgument("team"))
        .executes(CommandExecutor { _, args ->
            val team = args["team"] as Team

            // Toggle pvp
            team.setAllowFriendlyFire(team.allowFriendlyFire())
        })
        .register()
    // #endregion teamArgumentsExample
}

fun teamArgumentsDSL() {
    // #region teamArgumentsExampleDSL
    commandAPICommand("togglepvp") {
        teamArgument("team")
        anyExecutor { _, args ->
            val team = args["team"] as Team

            // Toggle pvp
            team.setAllowFriendlyFire(team.allowFriendlyFire())
        }
    }
    // #endregion teamArgumentsExampleDSL
}